include/ginkgo/core/multigrid/pgm.hpp

```diff
@@ -189,6 +190,11 @@ class Pgm : public EnableLinOp<Pgm<ValueType, IndexType>>,
 
     void generate();
 
+    std::tuple<std::shared_ptr<LinOp>, std::shared_ptr<LinOp>,
+               std::shared_ptr<LinOp>>
+    generate_local(
```

Comment:
I think it would make sense to add some documentation on what `generate_local `does and what the return type represent, to simplify understanding.